extensions/2.0/Khronos/KHR_spz_compression/README.md

```diff
@@ -0,0 +1,208 @@
+# KHR\_spz\_compression
+
+## Contributors
+
+- Jason Sobotka, Cesium
+- Adam Morris, Cesium
+- Sean Lilley, Cesium
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Adding Gaussian Splats](#adding-spz-compressed-data-to-primitives)
+  - [Extension Properties](#extension-properties)
+  - [Accessors](#accessors)
+- [Conformance](#conformance)
+- [Implementation](#implementation)
+- [Schema](#schema)
+- [Known Implementations](#known-implementations)
+- [Resources](#resources)
+
+## Overview
+

```

Comment:
At the start of the overview, we need to add a lot of context around Gaussian Splatting. We can probably reuse most if not all of the overview in the old document. Remember that this specification is not just adding SPZ support, but also enabling Gaussian Splatting within glTF.